src/NServiceBus.RavenDB/SagaPersister/SagaPersister.cs

```diff
@@ -245,7 +246,8 @@ static string DocumentIdForSagaData(IAsyncDocumentSession documentSession, Type
         static readonly Random random = new Random();
 
         readonly bool enablePessimisticLocking;
-        readonly int acquireLeaseLockRefreshMaximumDelayTicks;
+        readonly int acquireLeaseLockRefreshMaximumDelayMillisecondsWithoutFractions;
+        readonly int acquireLeaseLockRefreshMinimumDelayMillisecondsWithoutFractions;
```

Comment:
Personally the `WithoutFractions` isn't needed for my taste, that is obvious based on the `int` type.
```suggestion
        readonly int acquireLeaseLockRefreshMaximumDelayMilliseconds;
        readonly int acquireLeaseLockRefreshMinimumDelayMilliseconds;
```